src/lib.rs

```diff
@@ -187,9 +187,77 @@ macro_rules! make_guard {
         #[allow(unused)]
         let lifetime_brand = unsafe { $crate::LifetimeBrand::new(&branded_place) };
         let $name = unsafe { $crate::Guard::new(branded_place) };
+        if false {
+            // Prelude for poorman's specialization.
+            #[allow(unused)]
+            use $crate::__private::DefaultGet as __;
```

Comment:
Why the double-underscore? A single one is sufficient for anonymous trait import, right?

Reply:
Single leading underscore, in Rust, is typically used to silence an `unused` lint. Double is a better way to convey  "here be macro-related shenanigans"